### PR TITLE
Feature bloxinf 1266 highest att

### PR DESCRIPTION
**Pull request by vadiminc**

### CODE REVIEW

File: app/backend/services/account/account.service.ts [hunk 1]
```diff
@@ -38,9 +43,48 @@ export default class AccountService {
...
+      const generalData = await this.bloxApi.request(METHOD.POST, 'ethereum2/highest-attestation', payload);
+      const beaconchaData = await this.beaconchaApi.request(METHOD.GET, 'block/latest');
+      const keyManagerData = await this.keyManagerService.getAttestation();
+      console.warn('getHighestAttestation: raw answers', generalData, beaconchaData, keyManagerData);
+      logger.debug(`getHighestAttestation: raw answers > ${JSON.stringify({ generalData, beaconchaData, keyManagerData })}`);
+      Object.keys(generalData).forEach(key => {
+        const epoch = Math.max(...[
+          generalData[key].highest_source_epoch,
+          generalData[key].highest_target_epoch,
+          beaconchaData?.data?.epoch,
```
Comment (lior-blox) [inline app/backend/services/account/account.service.ts:63]: @vadiminc we should fail if there is no data from beaconcha or blox-api

File: app/backend/services/key-manager/key-manager.service.ts [hunk 2]
```diff
@@ -101,4 +101,22 @@ export default class KeyManagerService {
       throw new Error('Passphrase not correct');
     }
   }
+
+  @Catch({
+    showErrorMessage: true
+  })
+  async getAttestation(): Promise<any> {
+    try {
+      const { stdout: epochData } = await this.executor(`${this.executablePath} config current-epoch --network pyrmont`);
```
Comment (lior-blox) [inline app/backend/services/key-manager/key-manager.service.ts:110]: @vadiminc pyrmont hardcoded?

**Review by lior-blox — CHANGES_REQUESTED** [review]

File: app/backend/services/key-manager/key-manager.service.ts (hunk 2 above)
Comment (vadiminc) [reply app/backend/services/key-manager/key-manager.service.ts:110]: hmm you are right will fix

File: app/backend/services/account/account.service.ts (hunk 1 above)
Comment (vadiminc) [reply app/backend/services/account/account.service.ts:63]: if no data or request was failed?

Comment (vadiminc) [reply app/backend/services/account/account.service.ts:63]: I will add validator that 3 values should be existed

Comment (lior-blox) [reply app/backend/services/account/account.service.ts:63]: yea we must have data from atleast 2 sources, if u didn't get then fail

**Pull request merged**